.github/workflows/continuous-deployment.yml

```diff
@@ -132,6 +132,21 @@ jobs:
         run: |
           pnpm run --if-present test
 
+  visual-regression:
+    runs-on: ubuntu-latest
+    needs: install
+
+    steps:
+      - name: Publish to Chromatic
+        uses: chromaui/action@v11
+        if: |
+          github.event.pull_request.draft == false &&
+          github.actor != 'dependabot[bot]'
+        with:
+          autoAcceptChanges: main
+          projectToken: ${{ secrets.CHROMATIC_PROJECT_TOKEN }}
+          storybookBuildDir: packages/storybook/src/
```

Comment:
moet dit niet `packages/storybook/src/dist/` zijn?

Reply:
`packages/storybook/dist` zelfs :)

Reply:
Beide opties lijken niet te werken. Het blijkt een bekend probleem te zijn. Ik heb mijn case er ook al bij gezet: https://github.com/chromaui/chromatic-cli/issues/766#issuecomment-2110321181

Reply:
Klopt. Dit was een probeersel omdat de pipeline hier faalde. Nu bleek dat het probleem elders lag. Het is opgelost